### PR TITLE
[SD-3182] - https://simeoncloud.atlassian.net/browse/SD-3182

### PR DESCRIPTION
**Pull request by mzorzella-simeon**
Utilize a $friendlyName property in the CIS baseline configurations

### CODE REVIEW

File: Source/Resources/Content/MSGraph/Organization/Localizations/${ResourceContext%3ATenantId}--0.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+  },
+  "id": "0",
+  "loginPageLayoutConfiguration": {
+    "isFooterShown": false,
+    "isHeaderShown": true,
+    "layoutTemplateType": "default"
+  },
+  "organization-id": "${ResourceContext:TenantId}",
+  "usernameHintText": "user@${ResourceContext:TenantDomainName}",
+  "$friendlyName": null
```
Comment (Lance-Simeon) [inline Source/Resources/Content/MSGraph/Organization/Localizations/${ResourceContext%3ATenantId}--0.json:16]: Why null?

**Review by Lance-Simeon — COMMENTED** [review]
One last comment, I noticed these $friendlyNames are at the end of the file + spacing differences.  

My concern here is that M365 will want to reorder and respace the file after the first deploy, then we will be in the same scenario, where there are export changes to the files after the first deploy (the reording and respacing). @ashek-simeon  can you confirm if that will happen? If so, we will need to re-order (insert crying emogy here).

**Comment by ashek-simeon** [comment]
> One last comment, I noticed these $friendlyNames are at the end of the file + spacing differences.
> 
> My concern here is that M365 will want to reorder and respace the file after the first deploy, then we will be in the same scenario, where there are export changes to the files after the first deploy (the reording and respacing). @ashek-simeon can you confirm if that will happen? If so, we will need to re-order (insert crying emogy here).

I think that we should avoid reformatting all the files regardless of if M365 will care about it or not. In order to make this PR practical to review, I think prettymuch all files should have only change, an added line for friendly name.

I would suggest finding a way to use Rider to reformat all the files to their original format. If that doesn't work, then maybe we could find some other approach.

I would also suggest trying to use github copilot to write a quick script to put the properties in the json files back into alphabetical ordering (usually with $friendlyName right after $description)

File: Source/Resources/Content/AadIam/PasswordResetPolicies/Configuration.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-  "$description": "| What does this do? | Defines Azure's password reset policy and specifies the Azure AD groups to which it applies. This allows users to reset their passwords or unlock their Azure accounts. The baseline applies the password reset policy to members of the group \"Baseline - Microsoft 365 Users.\" |\n| Why should you use this? | If you want users to use self-service password reset instead of contacting IT support. This improves password reset security because it requires that users reset their passwords only via Mobile application code (the authenticator app) or SMS. |\n| What is the end-user impact? | <span style='color: red'>High Impact.</span> Users can securely recover and reset their passwords. |\n| Learn more | [How it works: Azure AD self-service password reset](https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-howitworks)",
-  "enablementType": 1,
-  "passwordResetEnabledGroupIds": [
-    "${urn:resource:MSGraph:Groups/Baseline - Microsoft 365 Users?id}"
-  ],
-  "passwordResetEnabledGroupName": "Baseline - Microsoft 365 Users"
-}
\ No newline at end of file
+{
+  "$friendlyName": "Configuration",
```
Comment (ashek-simeon) [inline Source/Resources/Content/AadIam/PasswordResetPolicies/Configuration.json:2]: As discussed, I think $friendlyName would be after $description alphabetically here

**Review by ashek-simeon — APPROVED** [review]

**Comment by ashek-simeon** [comment]
Thank you!!

**Review by Lance-Simeon — APPROVED** [review]

**Pull request merged**